src/core/client/ui/components/Popup/Popup.ts

```diff
@@ -1,5 +1,13 @@
 import { Component } from "react";
 
+interface WindowFeatures {
+  resizable: number;
+  menuBar: number;
```

Comment:
The browser API has it all lowercase:

```suggestion
  menubar: number;
```

Reply:
Fixed this up, also defaulted the value like we did previously with new naming.